docker/base/Dockerfile

```diff
@@ -10,4 +10,5 @@ RUN set -e ;\
     apt update -q=2 ;\
     apt install -q=2 --yes --no-install-recommends python3 python3-dev python3-pip python3-setuptools python3-wheel git build-essential libsnappy-dev ;\
     apt clean ;\
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* ;\
+    git clone https://github.com/vishnubob/wait-for-it.git /opt/wait-for-it
```

Comment:
Is it useful to lock this to a specific git hash?

Reply:
Good point.